### PR TITLE
Update whats-new-apps-19h1.md

### PR DESCRIPTION
**Pull request by msbemba**
Change in wordings # Snip & Sketch

### CODE REVIEW

**Comment by mypil** [comment]
@lizap - Can you please assist to check, approve and merge the content of this PR based on issue #181?

Thank you.

File: wip/at-home/whats-new-apps-19h1.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ We look forward to your feedback as we continue to test, learn, and improve the
 
 **3/6/2019 - Version 10.1901.10521.0**
 
-We’re in the process of rolling out out a new app update to Skip Ahead. It includes a number of fixes based on your feedback, among which are:
+We’re in the process of rolling out out a new app update to Snip & Sketch. It includes a number of fixes based on your feedback, among which are:
```
Comment (ItzLevvie) [inline wip/at-home/whats-new-apps-19h1.md]: There's duplicate "out" entries in "We’re in the process of rolling out out....."

**Comment by mypil** [comment]
@msbemba - please make the necessary typo corrections to this PR as suggested by @ItzLevvie. Thank you.

File: wip/at-home/whats-new-apps-19h1.md (hunk 1 above)
Comment (mypil) [inline wip/at-home/whats-new-apps-19h1.md]:
```suggestion
We’re in the process of rolling out a new app update to Snip & Sketch. It includes a number of fixes based on your feedback, among which are:
```

Comment (mypil) [reply wip/at-home/whats-new-apps-19h1.md]: Typo issue - please merge the commit.

**Comment by mypil** [comment]
@lizap - Can you please assist to check, approve and merge the content of this PR based on issue #181?

Thank you.

**Pull request merged**